pavelib/paver_tests/test_paver_quality.py

```diff
@@ -61,7 +61,7 @@ def test_pep8_parser(self):
 class TestPaverReportViolationsCounts(unittest.TestCase):
     """
     For testing utility functions for getting counts from reports for
-    run_jshint, run_complexity, run_safelint, and run_safecommit_report.
+    run_eslint, run_complexity, run_safelint, and run_safecommit_report.
```

Comment:
As mentioned in the other file, should we keep the JSHint tests until the feature is removed?